tools/block_cache_analyzer/block_cache_trace_analyzer.cc

```diff
@@ -1209,11 +1209,11 @@ void BlockCacheTraceAnalyzer::WriteBlockReuseTimeline(
   TraverseBlocks(block_callback);
 
   // A cell is the number of blocks accessed in a reuse window.
-  uint64_t reuse_table[reuse_vector_size][reuse_vector_size];
+  uint64_t* reuse_table = new uint64_t[reuse_vector_size * reuse_vector_size];
```

Comment:
Is it necessary to replace the stack space with heap? If so, it would be better to use unique_ptr to avoid potential memory leak.

Reply:
@zhichao-cao Agreed. I have made that change now.